src/scripts/playlists.js

```diff
@@ -1,203 +0,0 @@
-import loading from '../components/loading/loading';
```

Comment:
Why did this file become unnecessary?
How do we list playlists?

Reply:
I'm honestly not entirely sure... this has always been around but I don't think we have ever used it. Playlists are listed using the normal list controller. It's possible it was already dead at the time of the fork. 🤷‍♂️